### PR TITLE
🎨 Model pages revision

### PR DESCRIPTION
**Pull request by mamadOuologuem**
### Volunteers page 
Screen Shot 2019-11-20 at 2 12 01 PM

### Companies page 
Screen Shot 2019-11-20 at 2 12 30 PM

### Orga page 
Screen Shot 2019-11-20 at 2 12 19 PM

### Student page 
Screen Shot 2019-11-20 at 2 12 48 PM

### CODE REVIEW

**Comment by ericboucher** [comment]
- Let's be consistent with the number of projects on Ovio, 200->300
- Visit `Explore` now... -> let's have `Explore` in orange but not underlined
- Spread the word -> could we have a link to share?

- Submit | `viability` -> `sustainablity`
- How to Apply | `Request` -> `Request Info` or `Contact Us` - @tiphaineboyer thoughts?

**Comment by tiphaineboyer** [comment]
Yup, these changes look good to me!
Also, I know I owe a premade email for the request info. It's on my to do list.

**Comment by ericboucher** [comment]
So let's go for `Contact Us` and just `Email Us` instead of `Compose an Email`.

File: src/components/models/BlockHowToTemplate.js [hunk 1]
```diff
@@ -84,58 +95,68 @@ const BlockHowToTemplate = ({ title: blockTitle, subtitle, markImage, processIte
...
+                    ))}
+                  </Grid>
+                </Grid>
               </Grid>
             </Grid>
           );
         })}
       </Grid>
+      {blockButtonText && (
+        <Grid container item xs={6} justify="center">
```
Comment (ericboucher) [inline src/components/models/BlockHowToTemplate.js]: Maybe DRY the buttons a bit?

**Pull request merged**